miners/text/template_miner.py

```diff
@@ -55,6 +76,7 @@ def __init__(self, config ):
         # Local Model
         local_layers = TransformerEncoderLayer( bittensor.__network_dim__, self.config.nucleus.nhead, self.config.nucleus.nhid, self.config.nucleus.dropout )
         local_hidden_layers = TransformerEncoderLayer( bittensor.__network_dim__, self.config.nucleus.nhead, self.config.nucleus.nhid, self.config.nucleus.dropout )
+        # self.local_pos_encoder = PositionalEncoding(bittensor.__network_dim__, self.config.nucleus.dropout)
```

Comment:
Should the positional encoding be commented out?

Reply:
Sorry sorry, I think template_miner was pushed accidentally. It should not be pushed to this branch.